app/views/shared/_beta_banner.html.haml

```diff
@@ -1,6 +1,4 @@
 .flash.notice
-  .small This service is still in development, so for now only lists jobs in select schools in Cambridgeshire and the North East. When it launches, the full service will list jobs in schools throughout England.
+  .small This service is in development. It currently lists jobs in select schools in Cambridgeshire and the North East. When the service launches nationally, schools throughout England will be able to list jobs.
```

Comment:
in select**ed** schools

Reply:
Ran this past Fiona — she's happy with 'select'.